.claude/skills/skill-perfection/scripts/preflight.py

```
@@ -0,0 +1,249 @@
+            is_python = False
+        elif in_block and is_python:
+            block_code.append(line)
+
+    return blocks
+
+
+def check_python_syntax(code: str, line_start: int) -> Optional[Issue]:
+    """Check Python syntax using ast.parse."""
+    try:
+        ast.parse(code)
+        return None
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)